tests/test_education_crud.py

```diff
@@ -0,0 +1,32 @@
+import json
+from tests.BaseCase import BaseCase
+
+class TestEducationCrud(BaseCase):
+    def test_post_education_response(self):
+        # Given
+            # IN CASE WE NEED AUTHORIZATION (don't forget to update for our admin user):
```

Comment:
I'm sad this isn't working!

Reply:
Which part, the authorization or the entire test? The test **does** work! The authorization isn't built in because we don't have a user model yet, but I know it is coming!